Resources/public/js/components/news/form/main.js

```diff
@@ -0,0 +1,101 @@
+define(['underscore', 'jquery', 'text!./form.html'], function(_, $, form) {
+
+    return {
+
+        defaults: {
+            templates: {
+                form: form,
+                url: '/admin/api/news<% if (!!id) { %>/<%= id %><% } %>'
+            },
+            translations: {
+                title: 'public.title',
+                content: 'news.content'
+            }
+        },
+
+        header: {
+            title: 'news.headline',
+            toolbar: {
+                buttons: {
+                    save: {
+                        parent: 'saveWithOptions'
+                    }
+                }
+            }
+        },
+
+        layout: {
+            content: {
+                width: 'fixed',
+                leftSpace: true,
+                rightSpace: true
+            }
+        },
+
+        initialize: function() {
+            this.render();
+
+            this.bindDomEvents();
+            this.bindCustomEvents();
+        },
+
+        render: function() {
+            this.$el.html(this.templates.form({translations: this.translations}));
+
+            this.form = this.sandbox.form.create('#news-form');
+            this.form.initialized.then(function() {
+                this.sandbox.form.setData('#news-form', this.data || {});
```

Comment:
this line maps the given object with the help of `data-mapper-property` attributes